plugins/+testframework/GitHubLogTestPlugin.m

```diff
@@ -0,0 +1,16 @@
+classdef GitHubLogTestPlugin < matlab.unittest.plugins.TestRunnerPlugin

```

Comment:
Looking at the workflow results, it looks like this plugin might not be getting located. Does this have to do with the deployed version of scriptgen?

Reply:
Yeah, I'm trying to figure that out. It was working fine till I updated the name of the plugin to include `Test` as Sam suggested. Not sure how that affected it's functioning. Now, in the [logs](https://github.com/matlab-actions/run-tests/actions/runs/19859987162/job/56907318006#step:4:581) I see that the plugins folder is not able to be copied

 `cp: cannot stat './plugins': No such file or directory`

Reply:
Fixed now